src/sider/oppfolgingsplan/oppfolgingsplaner/BeOmOppfolgingsplan.tsx

```diff
@@ -26,10 +28,11 @@ import {
 import { useOppfolgingsplanForesporselDocument } from "@/hooks/oppfolgingsplan/useOppfolgingsplanForesporselDocument";
 import { oppfolgingstilfelle } from "../../../../test/aktivitetskrav/vurdering/vurderingTestUtils";
 import LabelAndText from "@/components/LabelAndText";
+import { useForm } from "react-hook-form";
 
 const texts = {
   aktivForesporsel:
-    "Obs! Det ble bedt om oppfølgingsplan fra denne arbeidsgiveren",
+    "Obs! Det ble bedt om oppfølgingsplan fra denne arbeidsgiveren", // TODO: Adjust to multiple arbeidsgivere ?
```

Comment:
Må vi endre ordlyden eller logikken rundt når denne vises når vi støtter flere arbeidsgivere?

Reply:
Jeg tenker det, og kanskje det etterhvert dukker opp et historikk-behov her?
Evt burde vi nok populere historikk-siden med dette ganske snart også 🤔 Må ikke glemme å holde den vedlike.

Reply:
Lurt med historikk ja :+1: 
Den aktiv forspørsel-advarselen vises jo nå hvis det finnes en forespørsel for det aktive oppfølgingstilfellet. Så kunne i først omgang bare endret teksten til det, evt vise navn på arbeidsgiveren det er forespurt fra.

Reply:
Tilpasset alert til å inkludere navn på virksomhet nå :+1: